addon/components/pix-input-password.hbs

```diff
@@ -0,0 +1,29 @@
+<div class='pix-input'>
+  {{#if @label}}
+    <label for={{this.id}} class="pix-input__label">{{@label}}</label>
+  {{/if}}
+  {{#if @information}}
+    <label for={{this.id}} class="pix-input__information">{{@information}}</label>
+  {{/if}}
+
+    <Input
+      id={{this.id}}
+      @type={{this.type}}
```

Comment:
chipotage

tu ne pourrais pas faire direct if this.isPasswordVisible 'text' 'password' ?

Reply:
je peut !